src/main/java/com/zontik/groshiky/controllers/AccountController.java

```diff
@@ -0,0 +1,39 @@
+package com.zontik.groshiky.controllers;
+
+import com.zontik.groshiky.model.Account;
+import com.zontik.groshiky.model.AccountResponse;
+import com.zontik.groshiky.service.IAccountService;
+import com.zontik.groshiky.service.IUserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+@Controller("/dashboard")
+public class AccountController extends BaseController {
+
+    private final IAccountService accountService;
+    private final IUserService userService;
+
+    @Autowired
+    public AccountController(IAccountService accountService, IUserService userService) {
+        this.accountService = accountService;
+        this.userService = userService;
+    }
+
+
+    @PostMapping(value = "/addAccount", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
+    public AccountResponse addAccount(Account account) {
+        Account newAccount = new Account();
```

Comment:
What do you need this newAccount for?

Reply:
I’ve replaced newAccount with a model, just to net return to the client an extra information.